torchci/pages/api/drci/drci.ts

```diff
@@ -293,7 +297,13 @@ export function constructResultsComment(
 
   let title = headerPrefix + icon + " " + title_messages.join(", ");
   output += title;
-  output += `\nAs of commit ${sha}:`;
+
+  output += `\nAs of commit ${sha} with merge base ${merge_base}`;
+  const timestamp = new Date(merge_base_date).valueOf() / 1000;
+  if (!isNaN(timestamp)) {
+    output += ` (<sub><sub><img alt="image" width=70 src="https://img.shields.io/date/${timestamp}?label=&color=FFFFFF&style=flat-square"></sub></sub>)`;
```

Comment:
Playing around with this site a bit, it appears to only work with integers.  The decimal it's being passed in will break the badges

Example:https://img.shields.io/date/1691751021.817

Reply:
added floor